managed/models/user_flags_helpers.go

```diff
@@ -22,24 +22,26 @@ import (
 	"gopkg.in/reform.v1"
 )
 
+// CreateUserParams has parameters to create a new user.
 type CreateUserParams struct {
 	UserID int
 }
 
+// UpdateUserParams has parameters to update existing user.
 type UpdateUserParams struct {
 	UserID int
 	Tour   bool
 }
 
+// CreateUser create a new user with given parameters.
 func CreateUser(q *reform.Querier, params *CreateUserParams) (*UserDetails, error) {
-	// TODO : Confirm if check is required
 	if params.UserID < 0 {
 		return nil, status.Errorf(codes.InvalidArgument, "Invalid user ID")
 	}
 
 	// Check user ID is unique
 	row := &UserDetails{ID: params.UserID}
-	if err := q.Reload(row); err != reform.ErrNoRows {
+	if err := q.Reload(row); errors.Is(err, reform.ErrNoRows) {
```

Comment:
something looks wrong here.
```suggestion
	if err := q.Reload(row); !errors.Is(err, reform.ErrNoRows) {
```